bsp/nuvoton/libraries/m031/rtt_port/drv_fmc.c

```diff
@@ -318,6 +318,7 @@ static int nu_fmc_init(void)
     SYS_LockReg();
 
     g_mutex_fmc = rt_mutex_create("nu_fmc_lock", RT_IPC_FLAG_PRIO);
+    RT_ASSERT(g_mutex_fmc != RT_NULL);
```

Comment:
assert是一种比较凶残的方式，直接让系统crash掉。有些不致命的问题可以直接返回错误和错误信息。

```suggestion
    if(g_mutex_fmc == RT_NULL)
    {
        LOG_E("mutex created failure");
        return (int)-RT_ERROR;
    }
```

在使用LOG_xxx时，该文件最上面也要定义LOG标签，参见：https://github.com/RT-Thread/rt-thread/blob/master/components/libc/compilers/common/ctime.c#L39-L41